charts/sourcegraph/templates/cadvisor/cadvisor.DaemonSet.yaml

```diff
@@ -119,6 +124,11 @@ spec:
       - name: disk
         hostPath:
           path: /dev/disk
+      {{- if .Values.cadvisor.containerSecurityContext.privileged }}
+      - name: kmsg
+        hostPath:
+          path: /dev/kmsg
+      {{- end }}
```

Comment:
there's no docker's `device` equivalent in `Kubernetes`, so we have to mount it as a regular volume.